interface/patient_file/summary/demographics.php

```diff
@@ -1034,14 +1034,6 @@ function setMyPatient() {
                     $pl = (AclMain::aclCheckIssue('medical_problem')) ? 1 : 0;
                     $meds = (AclMain::aclCheckIssue('medication')) ? 1 : 0;
                     $rx = (!$GLOBALS['disable_prescriptions'] && AclMain::aclCheckCore('patients', 'rx')) ? 1 : 0;
-                    $cards = $allergy + $pl + $meds + $rx;
-                    $col = "p-1 ";
-
-                    if ($cards > 0) {
-                        $colInt = 12 / $cards;
-                        $col = "col-" . $colInt;
-                    }
```

Comment:
Above was accounting for when user did not have access to all the issue categories
What happens if the user has access to medical_problem issues but not medication issues on the user interface?
(if looks a bit weird ok, just making sure no breakage)

Reply:
ahhh wondered what that was about i'll put back thanks

Reply:
I'm not going to try to persist top expand. Really should make it responsive to screen sizes with something like container-xl which will grow page when needed.
I never understood why you need to indicate with change expand icon whether an item is expanded or not! Umm just look at the page.